makejdk.sh

```diff
@@ -47,6 +47,8 @@ TARGET_DIR=""
 BRANCH=""
 KEEP=false
 JTREG=false
+JVM_VARIANT=""
```

Comment:
Do we have a list of these variants mentioned somewhere, our wiki or some README. I know these variants may vary a bit, good to have them to hand.